cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -2205,122 +2185,117 @@ features:
       success: You have successfully deployed the application "{appName}".
     ArtifactUpload:
       callToAction: Create a Pipeline
-      headerlabel: Add Third Party Driver
-      footertitle: Add Driver
+      headerlabel: Add third party driver
+      footertitle: Add driver
       Step1:
-        description: Upload your Driver
-        shorttitle: Upload Driver
+        description: Upload your driver
+        shorttitle: Upload driver
         title: Upload JAR
         uploadHelperText: Upload the third party driver that was downloaded in the previous step
         filePathLabel: Choose file
       Step2:
-        classnameLabel: Class Name
-        classnamePlaceholder: Driver Class Name. E.g. com.example.MyClass
+        classnameLabel: Class name
+        classnamePlaceholder: Driver class name. E.g. com.example.MyClass
         description: Configure the settings for your driver
         descriptionLabel: Description
-        decriptionPlaceholder: Driver Description
+        decriptionPlaceholder: Driver description
         nameLabel: Name
-        namePlaceholder: Driver Name
-        parentArtifactLabel: Parent Artifact
-        shorttitle: Driver Configuration
-        title: Configure Driver
+        namePlaceholder: Driver name
+        parentArtifactLabel: Parent artifact
+        shorttitle: Driver configuration
+        title: Configure driver
       success: You have successfully uploaded the driver "{artifactName}".
-      subtitle: You can now create a pipeline to extract data from Database using the driver.
+      subtitle: You can now create a pipeline to extract data from database using the driver.
     DirectiveUpload:
-      callToAction: Go to Preparation
-      footertitle: Upload Directive Artifact
+      callToAction: Go to preparation
+      footertitle: Upload directive artifact
       Step1:
-        description: Upload your Directive JAR.
+        description: Upload your directive JAR.
         errorMessage: Invalid directive. Directive must be a JAR file.
-        filePathLabel: Choose File
-        shorttitle: Upload Directive JAR
-        title: Upload Directive JAR
+        filePathLabel: Choose file
+        shorttitle: Upload directive JAR
+        title: Upload directive JAR
       Step2:
         description: Upload the directive configuration JSON.
         errorMessage: Invalid directive JSON. Plugin configuration should be in JSON format.
         errorMessageParentArtifacts: Invalid directive JSON. Please specify parent artifacts.
-        shorttitle: Upload Directive Configuration JSON
-        title: Upload Directive Configuration JSON
+        shorttitle: Upload directive configuration JSON
+        title: Upload directive configuration JSON
       subtitle: Start preparing data with the directive.
       success: You have successfully uploaded the directive "{pluginName}".
     Done: Done
     FailedMessage: Failed to {step}
-    GoToHomePage: Go to Homepage
```

Comment:
not needed?

Reply:
I didn't remove this, only changed it to `homepage`